src/chrome/komodo/content/pref/pref-syntax-checking.xul

```diff
@@ -147,6 +147,18 @@
                       </groupbox>
                     </vbox>
                 </groupbox>
+                <groupbox orient="vertical">
+                    <caption label="&eslintChecking.caption;"/>
+                    <description>
+                        &eslintChecking.description;
+                    </description>
+                    <hbox align="center">
+                         <checkbox id="lint_eslint_enabled"
+                            pref="true"
+                            preftype="boolean"
+                            label="&eslintChecking.checkbox;" />
+                    </hbox>
+                </groupbox>
```

Comment:
Sure, but like I said you cannot assume you're in the right directory to check in a node_modules folder. And the global `eslint` would be a fallback for if it can't find a local one and the pref is not set.